### DISCUSSION TITLE
Wrong date format exporting datatable in excel

### DISCUSSION THREAD

**Discussion by Alirzrezaei** [primefaces/community#1081] [PrimeFaces] [answered]
Hi,

since upgrading to PF13, date columns are output as numbers in Excel. In to PF 6, such columns were output as General format and thus as text. how can I fix to export with the expected format?

When my browser language is in English the format is OK but with German language wrong format is exported. 

Example:
Value in tatatable: 21.06.2023
Value in excel: 21.062.023

*** Apache Poi used for export to excel

Thank you very much
Alireza Rezaei

**Comment by melloware** [comment, marked as answer]
@Alirzrezaei can you try doing this: https://primefaces.github.io/primefaces/13_0_0/#/components/dataexporter?id=customization

set `stronglytypedCells` to false.

```java
excelOpt.setStronglyTypedCells(false);
```

It thinks your date is a NUMBER not a date so its setting the cell type to Numeric.
